tests/globalhelper/deployment.go

```diff
@@ -32,6 +33,18 @@ func IsDeploymentReady(operatorNamespace string, deploymentName string) (bool, e
 	return false, nil
 }
 
+// IsDeploymentInstalled checks if deployment is installed.
+func IsDeploymentInstalled(
```

Comment:
If you need this func only in affiliatedcertification test suite please move it under affiliatedcertification folder

Reply:
I think we will likely need this to test the operator suite in the future https://github.com/test-network-function/test-network-function/blob/main/CATALOG.md#operator